_summaries/salimans2016weight_1.md

```diff
@@ -1,19 +1,52 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{salimans2016weight}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: mmcinnestaylor 
+score: 7 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea  
+ * Salimans and Kingma propose Weight Normalization, a reparameterization of neural network weight vectors which they suggest leads to faster training via improving the conditioning of the optimization problem, and speeding up convergence of stochastic gradient descent.  
+ * This method is inspired by Batch Normalization, but unlike it, does not introduce dependencies amongst examples in a minibatch, thus (they claim) leading to better performance in applications such as recurrent models, deep reinforcement learning, and generative models where Batch Normalization is less successful. 
+ * They also claim that while Weight Normalization is simpler to implement than Batch Normalization, it realizes nearly the same speed-up, while also being computationally less expensive.
+## Technical Implementation
+ * The authors begin with standard neural network, where each neuron computes a weighted sum on input features followed by an elementwise nonlinearity. 
+   * text w is a k-dimensional  weight vector, b a scalar bias term, x is a k-dimensional bias term, &phi;() a nonlinearity     
+ y = &phi;(w &middot; x + b)  
+ * They then reparameterize the weight vectors, in terms of v, a k-dimensional vector, and g a scaling parameter  
+   * Equation 2 here  
+ * g may also be defined in terms of an exponential parameterization, where s is a log scale parameter learned by stochastic gradient descent  
+   * Scale Equation here  
+ * Training of the network used SGD, but the loss function now uses the reparameterizations above. Thus, the gradients in terms of those parameters are:  
+   * Equation 3 here
+   * Another formulation of the same gradients is:
+   * Equation 4 here
+   * This has the effect of scaling the weight gradient and projecting the gradient away from the current weight vector. The authors claim this brings the covariance matric of the gradient closer to the identity matrix, while also bebefiting optimization.
```

Comment:
bebefiting --> befitting

It might also be a good idea to mention that this is robust to different values of learning rates, and mention why that is so.